Libraries/Network/RCTHTTPRequestHandler.mm

```diff
@@ -63,6 +63,9 @@ - (NSURLSessionDataTask *)sendRequest:(NSURLRequest *)request
     callbackQueue.maxConcurrentOperationCount = 1;
     callbackQueue.underlyingQueue = [[_bridge networking] methodQueue];
     NSURLSessionConfiguration *configuration = [NSURLSessionConfiguration defaultSessionConfiguration];
+    [ configuration setHTTPShouldSetCookies:YES];
```

Comment:
Please, fix codestyle across all changed lines. This diff cannot be merged without it (because of linter).